<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroupWindow</class>
 <widget class="QDialog" name="AddGroupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>386</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Год поступления</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="boxYear"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Факультет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="boxFaculty">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Специальность</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="boxSpeciality"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Группа</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="boxGroup"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch_2">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Поиск по номеру</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchGroupText"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
